README.md

```diff
@@ -27,8 +27,8 @@ Compatible `Tracer` implementations:
 
 | Library | Status                     | Description |
 | ------- |----------------------------| ----------- |
-| [@slashmo](https://github.com/slashmo) / [**OpenTelemetry** Swift](https://github.com/slashmo/opentelemetry-swift) | Update for 1.0 in progress | Exports spans to OpenTelemetry Collector; **X-Ray** & **Jaeger** propagation available via extensions. |
-| [@pokrywka](https://github.com/pokryfka) / [AWS **xRay** SDK Swift](https://github.com/pokryfka/aws-xray-sdk-swift) | Not updated for 1.0        | ... |
+| [@slashmo](https://github.com/slashmo) / [**OpenTelemetry** Swift](https://github.com/slashmo/opentelemetry-swift) | 🟡 Update for 1.0 in progress | Exports spans to **OpenTelemetry Collector**; **Zipkin**, **X-Ray** and **Jaeger** compatible. |
```

Comment:
```suggestion
| [@slashmo](https://github.com/slashmo) / [Swift **OTel**](https://github.com/slashmo/swift-otel) | 🟡 Update for 1.0 in progress | Exports spans to [**OpenTelemetry Collector**](https://opentelemetry.io/docs/collector/); Compatible with **Zipkin**, **X-Ray** **Jaeger**, and more. |
```

Reply:
I wouldn't limit the OTel Collector to those three systems.

Reply:
Good call